### DISCUSSION TITLE
我想用scoop下载GitHub上的安装文件都走代理，如何设置

### DISCUSSION THREAD

**Discussion by Niefee** [MetaCubeX/mihomo#1365] [General]
![mx_screencap_20240705_160243](https://github.com/MetaCubeX/mihomo/assets/10347495/18dc414a-8a26-4bbd-b3f0-5f5256e3ab66)
 这样设置，然后点击添加前置规则，好像不起效，连接面板看不到相关日志

**Comment by Niefee** [comment]
补充一下，我用power shell，安装GitHub软件，也想走代理，好像不起效
安装pnpm的命令
`iwr https://get.pnpm.io/install.ps1 -useb | iex`

**Comment by minusmulticoloredper** [comment]
请开启Tun模式
